app/api/services/users/models.py

```diff
@@ -0,0 +1,24 @@
+import dataclasses
+from datetime import date
+
+from api.db.models import user_models
+from api.services.core import patch_params
+
+
+@dataclasses.dataclass
+class RoleParams:
+    type: user_models.RoleType
+
+
+@dataclasses.dataclass
+class CreateUserParams:
```

Comment:
I get organizing these service objects into a separate file, but also find it a bit annoying having to jump back and forth, not sure if there's a better way, any thoughts?

Reply:
good point. how about we put the objects next to the appropriate service? so, `CreateUserParams` lives in `create_user.py` and `PatchUserParams` lives in `patch_user.py`?

Reply:
That might be a bit clearer